astrix-test-support-junit5/src/main/java/com/avanza/astrix/test/AstrixExtension.java

```diff
@@ -137,11 +144,23 @@ public void afterAll(ExtensionContext context) {
     }
 
     public AstrixTestContext getAstrixTestContext(ExtensionContext context) {
+        final MapConfigSource configSource = getMapConfigSource(context);
         return getStore(context).getOrComputeIfAbsent(context.getRequiredTestClass(),
-                                                      key -> new AstrixTestContext(getConfiguration(context).getTestApis()),
+                                                      key -> new AstrixTestContext(
+                                                              configSource,
+                                                              getConfiguration(context).getTestApis()
+                                                      ),
                                                       AstrixTestContext.class);
     }
 
+    private MapConfigSource getMapConfigSource(ExtensionContext context) {
+        return getStore(context).getOrComputeIfAbsent(
+                MapConfigSource.class.getName(),
+                key -> new MapConfigSource(),
+                MapConfigSource.class
+        );
+    }
```

Comment:
Jag är osäker på effekterna av att spara `MapConfigSource` separat i `Store`n. Är det inte enklare att bara skicka in en ny då `AstrixTestContext` startas?
```java
    public AstrixTestContext getAstrixTestContext(ExtensionContext context) {
        return getStore(context).getOrComputeIfAbsent(context.getRequiredTestClass(),
                                                      key -> new AstrixTestContext(
                                                              new MapConfigSource(),
                                                              getConfiguration(context).getTestApis()
                                                      ),
                                                      AstrixTestContext.class);
    }
```

Reply:
Sure, we can do that. Have updated this in a separate commit now.